platform/linux-generic/odp_ipsec_sad.c

```diff
@@ -90,6 +90,13 @@ typedef struct sa_thread_local_s {
 	 * counter(s).
 	 */
 	odp_ipsec_op_status_t lifetime_status;
+
+	/*
+	 * Flag to check if the SA expiry status event was already
+	 * sent. This is applicable only for IPsec SA in inline outbound
+	 * mode.
+	 */
+	odp_bool_t status_notify;
```

Comment:
Why is this thread local? There is no need to send a separate status event for each thread that happens to notice soft limit crossing and making this thread local will consume more memory than having it as an atomic in the sa structure.

Reply:
Ack. Made status_notify as an atomic u32 in the SA structure with v5.